README.md

```diff
@@ -50,45 +56,68 @@ wget https://raw.githubusercontent.com/biocore/q2-chemistree/master/q2_chemistre
   --p-n-jobs 1 \
-  --o-collated-fingerprints collatedfp.qza
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-molecular-formulas MFs.qza
 ```
+This produces a QIIME 2 artifact of type `ZodiacFolder` with top-ranked molecular formula for MS1 features. Now, we predict molecular substructures in each feature based on the molecular formulas. We use [CSI:FingerID](https://www.pnas.org/content/112/41/12580) for this purpose as follows:
 
-This generates a contingency table with probabilities of various molecular substructures (i.e. molecular fingerprints) within an MS1 feature of type `FeatureTable[Frequency]`.
+```bash
+qiime chemistree predict-fingerprints --p-sirius-path 'sirius-osx64-4.0.1/bin' \
+  --i-molecular-formulas MFs.qza \
+  --p-ppm-max 20 \
+  --p-n-jobs 1 \
+  --p-java-flags "-Djava.io.tmpdir=/path-to-some-dir/ -Xms16G -Xmx64G" \
+  --o-predicted-fingerprints FPs.qza
+  ```
+This gives us a QIIME 2 artifact of type `CSIFolder` that contains probabilities of molecular substructures (total 2936 molecular properties) within in each feature.
+We now generate a contingency table with these probabilities i.e. molecular fingerprints of MS1 features in our experiment. This is of type `FeatureTable[Frequency]`.
 
-Now, we generate the molecular tree using:
+```bash
+qiime chemistree collate-fingerprint --i-csi-result FPs.qza \
+--p-qc-properties \
+--o-collated-fingerprints collated_FPs_QC.qza
+```
+By default, we only use PUBCHEM fingerprints (total 489 molecular properties). Replacing the flag `--p-qc-properties` by `--p-no-qc-properties` retains all the molecular properties in the contingency table. This table is used to generate out hierarchy of molecules!
 
 ```bash
 qiime chemistree make-hierarchy \
-  --i-collated-fingerprints collatedfp.qza \
+  --i-collated-fingerprints collated_FPs_QC.qza \
   --p-prob-threshold 0.5 \
   --o-tree demo-chemisTree.qza
 ```
 
-This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features. This is of type `Phylogeny[Rooted]`.
-
-Finally, note that SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances when using the `fingerprint` command). Thus, we need to remove the MS1 features without fingerprints from the feature table with:
+This generates a tree relating the MS1 features in these data based on molecular substructures predicted for MS1 features! This is of type `Phylogeny[Rooted]`. Note that SIRIUS predicts molecular substructures for a subset of features (typically for 70-90% of all MS1 features) in your experiment (based on factors such as sample type, the quality MS2 spectra, and used-defined tolerances such as `ppm-max`, `zodiac-threshold`). Thus, we need to remove the MS1 features without fingerprints from the feature table with:

```

Comment:
I would remove the exclamation mark at the end of the first sentence.

Can you change:
- `ppm-max` -> `--p-ppm-max`.
- `zodiac-threshold` -> `--p-zodiac-threshold`.